next-tavla/pages/api/organization/[oid]/members/index.ts

```diff
@@ -0,0 +1,42 @@
+import { verifyUserSession } from 'Admin/utils/auth'
+import {
+    getOrganizationById,
+    getUsersWithEmailsByUids,
+    removeUserFromOrganization,
+} from 'Admin/utils/firebase'
+import { concat } from 'lodash'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+export default async function handler(
+    request: NextApiRequest,
+    response: NextApiResponse,
+) {
+    const { oid } = request.query
+
+    const user = await verifyUserSession(request)
+    const organization = await getOrganizationById(oid as string)
+
+    const memberUids = concat(
+        organization.owners ?? [],
+        organization.editors ?? [],
+    )
+
+    if (!user || !memberUids.includes(user.uid)) {
+        return response.status(401).json({ message: 'Unauthorized' })
+    }
```

Comment:
Samme owner greia her

Reply:
Tanken med denne var vel at editors skulle se hvem andre som var i organisasjonen, og derfor trenger tilgang. Eller skal de ikke kunne se dette?

Reply:
Kan droppe det nå for MVPen egt